app/helpers/application_helper.rb

```diff
@@ -1,8 +1,15 @@
 module ApplicationHelper
   # @return [String]
   def navigation
-    govuk_header(service_name: 'Child development training', classes: 'noprint') do |header|
+    govuk_header(classes: 'noprint') do |header|
       header.navigation_item(text: 'Home', href: root_path)
+      header.custom_logo do
+        # rubocop:disable Style/StringConcatenation
```

Comment:
@agoldstone93 please use rubocop's todo list instead

Reply:
alternatively, can avoid doing it at all?

Reply:
For example could we relocate the contents of the `custom_logo` block to another `ApplicationHelper` method, that only uses tag helpers?